.github/workflows/build_nilrt.yml

```diff
@@ -25,6 +25,11 @@ jobs:
       with:
         access_token: ${{ github.token }}
 
+    - name: Set Swap Space
```

Comment:
The nilrt build has been borderline out-of-memory since I added the `-j2` flag. This is a way to get around that. TODO: change the @ to a commit SHA (or do something else).